maro/simulator/scenarios/data_center/business_engine.py

```diff
@@ -0,0 +1,320 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+from typing import Dict, List
+
+from yaml import safe_load
+
+from maro.event_buffer import AtomEvent, CascadeEvent, EventBuffer, MaroEvents
+from maro.simulator.scenarios.abs_business_engine import AbsBusinessEngine
+from maro.simulator.scenarios.helpers import DocableDict
+
+from .common import (
+    AssignAction, DecisionPayload, Latency, PostponeAction, PostponeType, ValidPhysicalMachine, VmFinishedPayload,
+    VmRequirementPayload
+)
+from .events import Events
+from .physical_machine import PhysicalMachine
+from .virtual_machine import VirtualMachine
+
+metrics_desc = """
+
+energy_consumption (int): Current total energy consumption.
+success_requirements (int): Accumulative successful VM requirements until now.
```

Comment:
success_schedulings, failed_schedulings

For total_energy_consumption, we need a formulation.

Can also add total_oversubscribtions, total_utilization (also need a formulation)

Reply:
What is the meaning of the `total_utilization` ? Since the `energy_consumption` is converted by the utilization, do we still need both?

Reply:
ok, total utilization is not correct, maybe min, max, mean, variance utilization in past (%)?